crypto/test_vectors/dump-test-vectors.sh

```diff
@@ -2,10 +2,11 @@
 
 set -e
 
-pushd ../../../../../crypto/proof-systems/poseidon/export_test_vectors
+DIR=$(pwd)
+cd ../../../../../crypto/proof-systems/poseidon/export_test_vectors
```

Comment:
Revert, use `pushd` and `popd` instead.

Reply:
this fails because it's run with `/bin/sh` and running with `/bin/bash` causes problems for other contributors: https://github.com/o1-labs/snarkyjs-bindings/pull/29

Reply:
Because of NixOS? The usual solution for that is `#!/usr/bin/env bash`  (or just not using NixOS) instead of falling all the way back to the Bourne Shell.

Reply:
nice, I can change it to that